pkg/sensors/program/loader_linux.go

```diff
@@ -692,13 +536,59 @@ func LoadUprobeProgram(bpfDir string, load *Program, maps []*Map, verbose int) e
 func LoadMultiKprobeProgram(bpfDir string, load *Program, maps []*Map, verbose int) error {
 	opts := &LoadOpts{
 		Attach: MultiKprobeAttach(load, bpfDir),
-		Open:   KprobeOpen(load),
-		Maps:   maps,
+		Open: func(coll *ebpf.CollectionSpec) error {
+			return nil
+		},
+		Maps: maps,
 	}
 	return loadProgram(bpfDir, load, opts, verbose)
 }
 
-func LoadFmodRetProgram(bpfDir string, load *Program, maps []*Map, progName string, verbose int) error {
+func LoadKProbeOverrideProgram(bpfDir string, load *Program, maps []*Map, progName string, verbose int, unloadFunc func(bool) error) error {
+	opts := &LoadOpts{
+		Attach: func(
```

Comment:
we tend to put callbacks code in functions, please add override attach and open fnuctions